src/prpy/util.py

```diff
@@ -749,3 +749,168 @@ def IsTimedTrajectory(trajectory):
     """
     cspec = trajectory.GetConfigurationSpecification()
     return cspec.ExtractDeltaTime(trajectory.GetWaypoint(0)) is not None
+
+
+def JointStatesFromTraj(robot, traj, times):
+    """
+    Helper function to extract the joint position, velocity and acceleration
+    from an OpenRAVE trajectory.
+    @param robot The OpenRAVE robot
+    @param traj An OpenRAVE trajectory
+    @param times List of times in seconds
+    @return pva_list List of list of position, velocity and acceleration
+                     at specified times. Inserts 'None' for unavailable fields
```

Comment:
This description doesn't make it clear that `pva_list` is a `list` of `list`s, where the `i`-th element is the `i`-th derivative of position.